### PR TITLE
Devkit maintenance - issue #38

### PR DESCRIPTION
**Pull request by MarkGreenPR**

### CODE REVIEW

File: devkit/bootstrap-unix.sh [hunk 1]
```diff
@@ -173,23 +225,27 @@ set -o errexit -o errtrace
...
     elif [ "${install_type}" == "pip" ]; then
+        echo ${salt_version}
         ${sudo} pip3 install ${install_name}==${salt_version}
     fi
     echo SaltStack ${install_type} installation complete
 fi
 
 echo -n "Version: "
 salt-call --version
+# exit
```
Comment (jscharber) [inline devkit/bootstrap-unix.sh:248]: probably dead code

File: devkit/bootstrap-unix.sh [hunk 2]
```diff
@@ -199,22 +255,24 @@ fi
...
+rm -rf ${tmpdir}
+${usersudo} git clone ${branch} https://github.com/pavedroad-io/pavedroad.git ${tmpdir}
 
 # Apply salt states
 echo Installing the development kit
-saltdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" &>/dev/null && pwd)
-${sudo} ${tmpdir}/devkit/apply-state.sh ${debug}
-mv ${tmpdir} ${saltdir}
+${usersudo} ${sudo} ${tmpdir}/devkit/apply-state.sh ${debug}
+# ${usersudo} ${sudo} ${tmpdir}/devkit/check-state.sh docker
```
Comment (jscharber) [inline devkit/bootstrap-unix.sh:264]: ? dead code

File: devkit/bootstrap-unix.sh [hunk 3]
```diff
@@ -199,22 +255,24 @@ fi
...
-    ${sudo} chown -R $(id -un):$(id -gn) ${homedir}
+if [ ${chown} ] ; then
+    ${sudo} chown -R ${bootuser}:${bootgroup} ${boothome}
 fi
 echo Development kit installation complete
 
-if command -v xdg-open >& /dev/null; then
+if [ ${DISPLAY} ] && command -v xdg-open >& /dev/null; then
     echo Opening the getting started page for the development kit
+    export BROWSER=/usr/bin/firefox
```
Comment (jscharber) [inline devkit/bootstrap-unix.sh:276]: probably should be a grain instead of hard coding firefox

**Comment by MarkGreenPR** [comment]
Same with apt-get install, left them in just in case we need them again.
Also I modify script on occasion for testing crap.

**Comment by MarkGreenPR** [comment]
Remember that we install Firefox, can't count on any other browser being
installed on new system.

File: devkit/bootstrap-unix.sh (hunk 1 above)
Comment (MarkGreenPR) [reply devkit/bootstrap-unix.sh]: done

File: devkit/bootstrap-unix.sh (hunk 2 above)
Comment (MarkGreenPR) [reply devkit/bootstrap-unix.sh]: done

File: devkit/bootstrap-unix.sh (hunk 3 above)
Comment (MarkGreenPR) [reply devkit/bootstrap-unix.sh:274]: As the devkit installs firefox it is the only browser guaranteed to be there for any new system across supported OSes and/or versions

**Pull request merged**